test/streams/broadcastable_test.rb

```diff
@@ -108,6 +108,65 @@ class Turbo::BroadcastableTest < ActionCable::Channel::TestCase
     end
   end
 
+  test "broadcasting action with attributes" do
+    assert_broadcast_on @message.to_gid_param, turbo_stream_action_tag("prepend", target: "messages", template: render(@message), "data-foo" => "bar") do
+      @message.broadcast_action "prepend", target: "messages", attributes: { "data-foo" => "bar" }
+    end
+  end
+
+  test "broadcasting action with no rendering" do
+    assert_broadcast_on @message.to_gid_param, turbo_stream_action_tag("prepend", target: "messages", template: nil) do
+      @message.broadcast_action "prepend", target: "messages", render: false
+    end
+  end
+
+  test "broadcasting action to with attributes" do
+    assert_broadcast_on "stream", turbo_stream_action_tag("prepend", target: "messages", template: render(@message), "data-foo" => "bar") do
+      @message.broadcast_action_to "stream", action: "prepend", attributes: { "data-foo" => "bar" }
+    end
+  end
+
+  test "broadcasting action to with no rendering" do
+    assert_broadcast_on "stream", turbo_stream_action_tag("prepend", target: "messages", template: nil) do
+      @message.broadcast_action_to "stream", action: "prepend", render: false
+    end
+  end
+
+  # The below commented tests pass if you change `Turbo::Streams::Broadcasts#broadcast_action_later_to`'s
+  # job call from `perform_later` to `perform_now`, but for some reason `perform_enqueued_jobs` is not running them.
+
```

Comment:
Wonder if I should leave these tests here in case anyone can fix them. But open to just removing them altogether.